### DISCUSSION TITLE
Schweizerdeutsch Phrases

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#86] [General]
Langenscheidt Lilliput

Rund 5000 Stichwörter und Wendungen

Amisant und originell for Unlauber und Freunde

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/203486130-e38fe915-9678-48ea-9e19-6b2068b7552b.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/203486198-1c6de1a9-acec-45bb-914b-409a832f78c9.jpeg)

**Comment by esthicodes** [comment]
mer gwöhnt sich dra
man gewöhnt sich daran

**Comment by iBeast666** [comment]
Wenn dich öpper ahaut
Wenn dich jemand, anspricht

**Comment by iBeast666** [comment]
Aamächelig: ankeifen
einführung in programmiereung

am arsch isch finschter!
Aasäiche : (vulgär) beschimpfen
Aaschiisse: anwidern, langweilen;
Das hät mi so aagschisse: das hat mir derart gestunken (darauf hatte ich gar keine Lust)
Aaschlaa: sich stoßen;
Ich han s Chnüü aagschlage
Ich habe mir das Knie angeschlagen/gestossen

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/203502105-bf1ae10c-45d4-43db-a265-a2d94d7b14ea.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/203502161-57352ed8-4444-4c1e-b94d-f5e192c3d19a.jpeg)

**Reply by Ethgrossesxheisse** [reply]
Ja sicher, stärnäföifi

**Reply by Ethgrossesxheisse** [reply]
Es het em de ermel ie gno

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/203867457-1a7c3e27-e2cb-453a-9428-c562584d825b.jpeg)

**Reply by esthicodes** [reply]
Adjöö!
- auf Wiedersehen!

Aff: 
-

**Comment by esthicodes** [comment]
A:

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/204089890-4dab590a-4cdb-4476-876a-49daea01cf6a.jpeg)

![image](https://user-images.githubusercontent.com/78131082/204089893-a010c2fe-830c-4635-ab98-0baeb3fa0b29.jpeg)

**Comment by esthicodes** [comment]
äierschwämmli s pfefferling
Äiertätsch de Eierkuchen, Omelett
Äifränkler de Einfrankenstück
Äine de Einer, eine 
Me äis (=es gibt mir eins)

**Comment by esthicodes** [comment]
zum Geburtstag ein paar
Beauty Tricks 

**beseitigt Altersflecken**
- was? 
- das Bedeutet das es flecken die vom Alterverursacht werden entfernt werden😏

**beseitigt Fältchen**
- was? 🥺
- es macht schönheiss flecken weg die man nicht haben will zum beistpiel wartzen oder verunreinigtheiten oder ander sachen die man nicht haben will

**beseitigt Krähenfüsse**
 das sind kleine Grübchen bei den augen 👵🏻🦉🐨🪳

**beseitigt schlaffe Haut**
- macht die haut wieder straff (glatt sanft Ich wette, Sie lösen Kreuzworträtsel mit Tinte. 

**beseitigt Tränensäcke**
- was bedeutet das?

![image](https://user-images.githubusercontent.com/78131082/204135289-d55472d2-67ea-4566-b03f-3ec1778f8901.png)


**beseitigt Augenringe**

- was bedeutet das?

![image](https://user-images.githubusercontent.com/78131082/204135404-def37e9c-1fe5-4aa2-b20d-0dcd591dc8c2.jpeg)


- _ Kaltes Wasser: Einfach, aber ebenfalls wirksam: Lege ein paar Abschminkpads in kaltes Wasser und lege sie auf deine Augen. Lieber keine Teebeutel: Oft wird empfohlen, verwendete Teebeutel abkühlen zu lassen und sie dann für zehn bis 15 Minuten auf die Augen zu legen_

**Comment by ghost** [comment]
Mein Lieblingsteil Tag ist weihnachten wenn wir ihn mit der ganzen Familie feiern

Was isch din lieblings frei tag?

**Comment by ghost** [comment]
Weihnachten, will mir mit de ganze familie feschte

**Comment by ghost** [comment]


**Reply by ghost** [reply]
marry x-mas

**Comment by ghost** [comment]


**Reply by ghost** [reply]
dihei si

**Comment by ghost** [comment]


**Reply by ghost** [reply]
???
